src/schema/validator.js

```diff
@@ -380,7 +380,7 @@ export class SchemaValidator {
     if (!this._localeValidator) {
       this._localeValidator = this._validator.compile({
         ...this.messagesSchemaObject,
-        $id: 'messages',
+        $id: 'i18nMessages',
         $ref: '#/types/WebExtensionMessages',
```

Comment:
```suggestion
        $ref: '#/types/WebExtensionI18nMessages',
```

Let's also rename the type, as well as the definition of this ID in the (newly renamed) i18nMessages.json file. Note: the `I18n` is the uppercase letter I (before J), not lowercase L (before M).

Reply:
Done.